dist/index.js

```diff
@@ -317,6 +317,7 @@ function runAction() {
                 yield sleep(constants.DEFAULT_EXECUTION_GRAPH_CHECK_INTERVAL);
                 executionGraph = yield getExecutionGraph(executionGraphId);
             }
+            core.startGroup("");
```

Comment:
So, according to the screenshot, you need to put a message here. Otherwise it shows an empty ">"

Something like "Uploading artifacts..."